modules/apps/object/object-test/src/testFunctional/macros/ObjectRelationships.macro

```diff
@@ -0,0 +1,23 @@
+definition {
+
+	macro assertRelationshipParameterNotPresent {
+		Click(locator1 = "ObjectRelationships#SELECT_PARAMETER_FORM");
+
+		AssertElementNotPresent(
+			key_parameterOption = "${parameter}",
+			locator1 = "ObjectRelationships#SELECT_PARAMETER_OPTION");
+	}
```

Comment:
We can remove this macro and add the functions to the test level. I want us to start moving our assertions out of macros, I'll explain more next week.